src/Dapr.Actors/Client/ActorProxyFactory.cs

```diff
@@ -14,6 +14,7 @@
 namespace Dapr.Actors.Client
 {
     using System;
+    using System.Globalization;
```

Comment:
Since we're here, do you mind moving these imports outside of the namespace?

Reply:
Sure!